server/notifications/message/plain.go

```diff
@@ -0,0 +1,29 @@
+package message
+
+import (
+	"fmt"
+
+	"github.com/Hukyl/genesis-kma-school-entry/models"
+)
+
+// PlainRateMessage is a message that contains the exchange rate between two currencies.
+type PlainRateMessage struct {
+	rate *models.Rate
+}
+
+func (m *PlainRateMessage) SetRate(rate *models.Rate) {
+	m.rate = rate
+}
+
+func (m *PlainRateMessage) Subject() string {
+	return fmt.Sprintf("%s-%s exchange rate", m.rate.CurrencyFrom, m.rate.CurrencyTo)
+}
+
+func (m *PlainRateMessage) String() string {
+	return fmt.Sprintf(
+		"1 %s = %f %s",
+		m.rate.CurrencyFrom,
+		m.rate.Rate,
```

Comment:
maybe it's better to format float with some precision here?
I.E `"1 %s = %.2f %s",` will produce 42.2342453 -> 42.23

what do you think?

Reply:
exact rate is quite often more useful for precise calculations